common/test/acceptance/pages/studio/import_export.py

```diff
@@ -10,6 +10,19 @@
 from . import BASE_URL
 
 
+class TemplateCheckMixin(object):
```

Comment:
I don't think it would be *that* bad within six months. It may take a moment longer, but my IDE would easily be able to trace the source of that property. If a test doesn't show up on a class itself, or a property doesn't show up on a class itself, it makes sense that it's on one of the superclasses. As the test is properly labeled as 'test_template', it should be obvious which superclass is relevant, both on the test and page side, I would think.

I suppose I'm biased on that matter, because I wrote it, but this feels like the right way to do it from a composition perspective.